deployment/docker-compose.yml

```diff
@@ -7,152 +7,162 @@
 #
 # See accompanying Make commands for easy collectstatic etc.
 
-smtp:
-  image: catatnight/postfix
-  hostname: postfix
-  environment:
-    # You could change this to something more suitable
-    - maildomain=kartoza.com
-    - smtp_user=noreply:docker
-  restart: unless-stopped
+version: '2'
```

Comment:
Move on to version 2 based config.

Reply:
what's the difference between version 2 and 3?

Reply:
v2 to v3 has huge feature differences.
docker-compose v3 is more declarative, because they want to use it as cluster orchestration.
This file uses compose v1 (before I changed it), then we changed it to v2.
The reason we didn't change it to v3 (yet) is because our Rancher server is using Rancher v1.6, which only supports compose v2.
By using compose v2, we reduce the hassle when migrating the compose file to Rancher catalog.

Reply:
Great, thank you for the explanation 👍